apps/cli/src/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { dirname } from "node:path";
```

Comment:
The whole file has different error catching strategy.
- `deleteFolderIfExists` catches rm and then just console that `Error while deleting the folder`

others do not catch 
`await writeFile`
`await mkdir`
And use try/catch for logic.

To make consistent I will re-throw error at `deleteFolderIfExists` instead of `console.error("Error while deleting the folder: \n", err);`